functional-tests/steps/when.ts

```diff
@@ -0,0 +1,132 @@
+import { When } from "cucumber";
+
+import { openWebsite } from "@nice-digital/wdio-cucumber-steps/lib/support/action/openWebsite";
+import { clickElement } from "@nice-digital/wdio-cucumber-steps/lib/support/action/clickElement";
+import { waitFor } from "@nice-digital/wdio-cucumber-steps/lib/support/action/waitFor";
+import { checkIfElementExists } from "@nice-digital/wdio-cucumber-steps/lib/support/lib/checkIfElementExists";
+
+import { typeInSearchBox } from "../support/action/typeInSearchBox";
+import { scrollInToView } from "../support/action/scrollInToView";
+import { waitForTitleToChange } from "../support/action/waitForTitleToChange";
+import { waitForUrlToChange } from "../support/action/waitForUrlToChange";
+import { waitForScrollToElement } from "../support/action/waitForScrollToElement";
+import { acceptCookieBanner } from "../support/action/acceptCookieBanner";
+import { waitForReact } from "../support/action/waitForReact";
+import { waitForSearchLoad } from "../support/action/waitForSearchLoad";
+import { getSelector } from "../support/selectors";
+import { getPath } from "../support/pagePaths";
+
+When(/^I type "([^"]*)" in the header search box$/, typeInSearchBox);
+
+When(
+	/^I click "([^"]*)" in the autocomplete options$/,
+	async (text: string) => {
+		const pageTitle = await browser.getTitle();
+		await typeInSearchBox("Ast");
+
+		const optionElement = await $(await getSelector("autocomplete option"));
+		await optionElement.waitForExist();
+
+		const anchorSelector = await getSelector("autocomplete anchor");
+
+		// For some reason we can't click on an autocomplete suggestion via wdio's
+		// browser.click(element). So we have to use this workaround:
+		await browser.execute(
+			(text, optionAnchorSelector) => {
+				document.querySelectorAll(optionAnchorSelector).forEach((element) => {
+					if (
+						element.textContent &&
+						element.textContent.toLowerCase().indexOf(text.toLowerCase()) > -1
+					) {
```

Comment:
Is this missing an `else` statement or is it short hand?

Reply:
It just doesn't need an else